tailwind.config.ts

```diff
@@ -8,5 +9,17 @@ const config: Config = {
     "./plugins/**/*.{js,ts,jsx,tsx,mdx}",
     "./node_modules/@aragon/ods/**/*.js",
   ],
+  theme: {
+    extend: {
+      colors: {
+        voting: {
+          yes: "#10b981",
+          no: "#dc2626",
+          abstain: "#94a3b8",
+          slate: "#94a3b8",
+        },
+      },
+    },
+  },
```

Comment:
added this as not all the tailwind class are working but using custom one defined in `@aragon/ods/tailwind.config` with just a subset